packages/core/src/external/fhir/shared/utils.ts

```diff
@@ -1,26 +1,38 @@
 import { Bundle, Resource } from "@medplum/fhirtypes";
 import { dangerouslyDeduplicateFhir } from "../../../fhir-deduplication/deduplicate-fhir";
 import { normalizeFhir } from "../normalization/normalize-fhir";
-import { buildCollectionBundle } from "./bundle";
+import { buildBundle, RequiredBundleType } from "./bundle";
+import { MetriportError } from "@metriport/shared";
+import { cloneDeep } from "lodash";
 
-export const mergeBundles = ({
+export function mergeBundles({
   cxId,
   patientId,
   existing,
   current,
+  bundleType,
 }: {
   cxId: string;
   patientId: string;
   existing: Bundle<Resource>;
   current: Bundle<Resource>;
-}) => {
+  bundleType: RequiredBundleType;
+}): Bundle<Resource> {
   if (!existing.entry || !current.entry) {
-    throw new Error(
-      `Entry field on bundles must exist: existing: ${!!existing.entry} current: ${!!current.entry}`
+    throw new MetriportError(
+      `Entry field on bundles must exist: existing: ${!!existing.entry} current: ${!!current.entry}`,
```

Comment:
Can we keep the error message static, please? It helps grouping them in Sentry and we have that info on the `additionalInfo` param.